services/spar/src/Spar/SCIM.hs

```diff
@@ -271,11 +272,17 @@ instance SCIM.UserDB Spar where
 
     buid <- Id <$> liftIO UUID.nextRandom
     -- TODO: Assume that externalID is the subjectID, let's figure out how
-    -- to extract that later. The question of "who's the issuer?" is a very
-    -- tricky one and we don't have a decision here yet.
-    let uref = SAML.UserRef
-          (SAML.Issuer $ SAML.unsafeParseURI ("https://unknown.issuer.org"))
-          (SAML.opaqueNameID extId)
+    -- to extract that later
+    -- TODO: when the issuer is deleted, the token still remains, so we can
+    -- fail here
```

Comment:
this should be fine, no?  just throw an appropriate error.  please resolve this TODO!

Reply:
NB: Ideally we should delete the token when the issuer is deleted, but maybe that's hard to pull off and that's not done in this PR.

Reply:
> NB: Ideally we should delete the token when the issuer is deleted, but maybe that's hard to pull off and that's not done in this PR.

what do you mean by `Issuer`?  the idp?  then what about idp-less tokens?

i thought that tokens should last as long as idp (if there is one) or team (if there isn't), or if they are deleted manually.

1. with idp: just add a line or two to the `Data.deleteIdP` (or whatever it's called).
2. without idp: oh, right.  i asked you to move that to #519.  i guess this TODO should also be resolved there, then.  :-)
3. manually: already done.

Reply:
> with idp: just add a line or two to the Data.deleteIdP (or whatever it's called).

Done.

> without idp

Yeah, will be done in #519.